nodecontrol/kmdControl.go

```diff
@@ -192,7 +192,7 @@ func (kc *KMDController) StartKMD(args KMDStartArgs) (alreadyRunning bool, err e
 			return false, errors.New("bad kmd data dir")
 		}
 		if (dataDirStat.Mode() & 0077) != 0 {
-			logging.Base().Errorf("%s: kmd data dir exists but is too permissive (%o)", kc.kmdDataDir, dataDirStat.Mode()&0777)
+			logging.Base().Errorf("%s: kmd data dir exists but is too permissive (%o), change to (%o)", kc.kmdDataDir, dataDirStat.Mode()&0777, dataDirStat.Mode()&0077)
```

Comment:
I believe that you meant to have the following on the second one, right ? 
`dataDirStat.Mode()^0077`

Reply:
"_dataDirStat.Mode()&0077_"  can be replaced with "_dataDirStat.Mode()&0644_" to show the right permissions. Just a suggestion.

Reply:
Thank you for the suggestions, this should be fixed now.